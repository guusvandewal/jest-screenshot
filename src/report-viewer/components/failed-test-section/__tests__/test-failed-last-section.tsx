import * as React from "react";
import { shallow } from "enzyme";
import { FailedTestSection } from "..";

describe("FailedTestSection", () => {
    it("looks as expected", () => {
        expect(shallow(
            <FailedTestSection
                failedTest={testResults.files[0].failedTests[0]}
            />,
        )).toMatchSnapshot();
    });
});
