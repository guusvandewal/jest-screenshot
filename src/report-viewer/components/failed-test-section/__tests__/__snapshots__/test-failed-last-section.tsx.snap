// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FailedTestSection looks as expected 1`] = `
<section
  className="section"
>
  <p
    className="subtitle is-4"
  >
    The thing with this other thing does things
  </p>
  <Snapshot
    key="The first snapshots-1"
    snapshot={
      Object {
        "changedPixels": 5,
        "changedRelative": 0.5,
        "diffPath": "jest-screenshot-report/reports/some-identifier.snap.png/diff.png",
        "height": 2,
        "message": "Received message.",
        "receivedPath": "jest-screenshot-report/reports/some-identifier.snap.png/received.png",
        "snapshotNumber": 1,
        "snapshotPath": "jest-screenshot-report/reports/some-identifier.snap.png/snapshot.png",
        "testFileName": "src/__tests__/test-first.ts",
        "testName": "The first snapshots",
        "totalPixels": 10,
        "width": 5,
      }
    }
  />
  <Snapshot
    key="The first snapshots-2"
    snapshot={
      Object {
        "changedPixels": 20,
        "changedRelative": 0.2,
        "diffPath": "jest-screenshot-report/reports/some-other-identifier.snap.png/diff.png",
        "height": 10,
        "message": "Received message.",
        "receivedPath": "jest-screenshot-report/reports/some-other-identifier.snap.png/received.png",
        "snapshotNumber": 2,
        "snapshotPath": "jest-screenshot-report/reports/some-other-identifier.snap.png/snapshot.png",
        "testFileName": "src/__tests__/test-first.ts",
        "testName": "The first snapshots",
        "totalPixels": 100,
        "width": 10,
      }
    }
  />
</section>
`;
